src/main/java/roomescape/service/ReservationTimeService.java

```diff
@@ -0,0 +1,63 @@
+package roomescape.service;
+
+import java.util.List;
+import org.springframework.stereotype.Service;
+import roomescape.domain.reservation.Reservation;
+import roomescape.domain.reservation.ReservationRepository;
+import roomescape.domain.time.Time;
+import roomescape.domain.time.TimeRepository;
+import roomescape.dto.reservation.ReservationRequest;
+import roomescape.dto.reservation.ReservationResponse;
+import roomescape.dto.time.TimeRequest;
+import roomescape.dto.time.TimeResponse;
+
+@Service
+public class ReservationTimeService {
+
+    private final ReservationRepository reservationRepository;
+    private final TimeRepository timeRepository;
+
+    public ReservationTimeService(ReservationRepository reservationRepository, TimeRepository timeRepository) {
+        this.reservationRepository = reservationRepository;
+        this.timeRepository = timeRepository;
+    }
+
+    public List<ReservationResponse> findAllReservations() {
+        return reservationRepository.findAll()
+                .stream()
+                .map(reservation -> ReservationResponse.from(reservation))
+                .toList();
+    }
+
+    public ReservationResponse createReservation(ReservationRequest reservationRequest) {
+        Long timeId = reservationRequest.timeId();
+        Time time = timeRepository.findById(timeId);
```

Comment:
```suggestion
        Time time = timeRepository.findById(reservationRequest.timeId());
```
개인적으로 이정도 변수는 넣어주는 편인데, 이건 땡이의 스타일에 따라 수정할지 말지 경정해주세요!

Reply:
단순히 getter를 호출해주는 정도는 변수로 분리하지 않아도 괜찮을 것 같습니다 👍